### ISSUE TITLE
Install 0.5.3 Debian 11

### ISSUE THREAD

**Issue by wouam31200** [friendly-bits/geoip-shell#11]
Hi, I wanted to update the script.
I was on version 0.2.7.
I uninstalled 0.2.7 and installed 0.5.3 except I got this error.
If I reinstall 0.2.7 everything works fine.
I'm on Debian 11.
Thank you so much.

```
Creating the data dir '/var/lib/geoip-shell'... Ok.
manage: Applying config...
Deleting the data directory '/var/lib/geoip-shell'...

Checking connectivity... fetch: Error: Machine appears to have no Internet connectivity or RIPE server is currently down.
run: Error: Failed to fetch and validate lists 'FR_ipv4 FR_ipv6'.
run: Warning: Aborting the action 'add'.

manage: Error: Failed to verify ip lists coherence.
manage: Firewall ip lists: ''
manage: Config ip lists: 'FR_ipv4 FR_ipv6'
manage: Warning: Discrepancy detected between the firewall state and the config file.
manage: Warning: missing ip lists in the firewall: 'FR_ipv4 FR_ipv6'
manage: Error: Failed to apply geoip-shell config.
manage: Warning: Discrepancy detected between the firewall state and the config file.
manage: Warning: missing ip lists in the firewall: 'FR_ipv4 FR_ipv6'
install: Error: Failed to create and apply the iplist.
```

**Comment by friendly-bits** [comment]
Hi, thanks for reporting. The current version relies on the nslookup utility for connectivity check and turns out that some distributions don't ship it by default. I'll fix this soon - in the meanwhile, please install package 'dnsutils'. Please let me know if you have further issues.

**Comment by wouam31200** [comment]
Great!
It works by installing 'dnsutils'.

On the other hand, I have a little problem, in iptables I authorized 2 IP addresses (which are not part of the countries authorized in the white list)
So, when I install geoip-shell, the two addresses are no longer authorized.
Is there a way to bypass the blockage?
Thank you very much

**Comment by friendly-bits** [comment]
Did you try the `geoip-shell configure -t` command as described in the Usage section of the README?
https://github.com/friendly-bits/geoip-shell?tab=readme-ov-file#usage

**Comment by wouam31200** [comment]
I'm sorry, I didn't watch.
If I understand correctly, after installation I do:
geoip-shell configure -t <["ip_addresses1"]|["ip_addresses2"]>
to unblock both IP addresses.

**Comment by friendly-bits** [comment]
Here's an example: `geoip-shell configure -t "1.1.1.1 2.2.2.2"`

If you don't have the current version installed yet, you can achieve the same if specify trusted ip's during installation: `sh geoip-shell-install.sh -t "1.1.1.1 2.2.2.2"`

**Comment by wouam31200** [comment]
Thank you very much for your help.
You did a great job!
